cerebrl/forgerock-web-login-framework
# feat(suspended-id): add support for suspended id

## Summary

Adds support for the Email Suspend Node and "journey continuation" after the clicking of a "magic link".

## Details

- Support for `SuspendedTextOutputCallback` already existed
- Implement support for handling `suspendedId` from "magic link"
- Fix issue with `authIndexValue` being appended to `/authenticate` when sending `suspendedId`
- Fixing app and e2e test implementation that set a default tree on initialization (having a `tree` value on the base config makes it difficult to handle journey continuation)
- Added documentation for new APIs as well as tutorial

## Manual test

https://forgerock-web-login-framework-git-featsuspended-emai-d87688-cix.vercel.app/?journey=ResetPassword

Make sure to test with a user that has a valid email address you can access. Check your email and just copy the parameters portion of the link (without clicking on it). The paste the parameters replacing the existing parameters on the above URL. 

For example (don't actually use):

https://forgerock-web-login-framework-git-featsuspended-emai-d87688-cix.vercel.app/?realm=/alpha&suspendedId=Ii4yP7fSeaq2N4SZsdnoQ45JYqI
